Sources/NIO/SocketChannel.swift

```diff
@@ -412,6 +411,21 @@ final class DatagramChannel: BaseSocketChannel<Socket> {
             #else
             break
             #endif
+        case _ as ChannelOptions.Types.ExplicitCongestionNotificationsOption:
+            let valueAsInt: Int32 = value as! Bool ? 1 : 0
+            switch self.localAddress?.protocol {
+            case .some(.inet):
+                try self.socket.setOption(level: NIOBSDSocket.OptionLevel.ip,
+                                          name: NIOBSDSocket.Option.ip_recv_tos,
```

Comment:
Can we use the short names for these? `setOption(level: .ip, name: .ip_recv_tos, value: valueAsInt)`.